common/rtc_manager_rviz_plugin/README.md

```diff
@@ -0,0 +1,21 @@
+# rtc_manager_rviz_plugin
+
+## Purpose
+
+This plugin displays the current status of autoware.  
+This plugin also can engage from the panel.
```

Comment:
Could you write the purpose of this plugin?

Reply:
@rej55 
I fixed at 
fbf410b